### DISCUSSION TITLE
Pop-ups with multiple Entries?

### DISCUSSION THREAD

**Discussion by mprogers** [dotnet/maui#10489] [Q&A] [answered]
What if I want to get more than one piece of information from a user? There must be someway to do that, obviously but DisplayPrompt() only seems to have one field. So, what's the recommended way to do this? What I want is to be able to mimic iOS behavior - when I tap on a button, a window slides modally into view with multiple entries - and when I dismiss them that data gets returned.

I didn't see any discussion of this in the .NET MAUI UI section ...

**Comment by skenuo** [comment, marked as answer]
use .NET MAUI Community Toolkit Popup
https://learn.microsoft.com/en-us/dotnet/communitytoolkit/maui/views/popup
